src/extension/utils/toolkit.js

```diff
@@ -123,3 +123,14 @@ export function addToolkitEmberHook(context, componentKey, lifecycleHook, fn) {
 
   ynabToolKit.hookedComponents.add(componentKey);
 }
+
+export function removeToolkitEmberHooks(context, componentKey, lifecycleHook) {
+  const componentProto = Object.getPrototypeOf(componentLookup(componentKey));
+
+  let hooks = componentProto[emberComponentToolkitHookKey(lifecycleHook)];
+  if (hooks) {
+    componentProto[emberComponentToolkitHookKey(lifecycleHook)] = hooks.filter(
```

Comment:
this would technically work but I think honestly I'd rather just filter the function itself since thats more traditional for things like `on/off` patterns

we shouldn't need to pass in `this`, just do 

`removeToolkitEmberHooks(componentKey, lifecycleHook, <function>)`